### DISCUSSION TITLE
Cannot register solokey USB-A

### DISCUSSION THREAD

**Discussion by renaclerican35** [solokeys/solo2#119] [Q&A] [answered]
Hi !

I cannot register my solokeys on the website solokeys.com. I don't find the security settings.
Can you help me ?
In advance, I thank you for your answer.

**Comment by needs-coffee** [comment, marked as answer]
what do you mean register on solokeys.com? 
You register FIDO authenticators on any website that you want to use it with such as google or github.
Solokeys website is just telling you how to register the key on the website you want to use it with.
